appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -192,7 +204,7 @@ public void resetMetricsSecureProperty() {
         MetricRegistry registry = getRegistry(registryName);
         Map<String, Metric> metricMap = registry.getMetrics();
         if (metricMap.containsKey(metricName)) {
-            return singletonMap(metricName, metricMap.get(metricName));
+            return Collections.singletonMap(metricName, metricMap.get(metricName));
```

Comment:
Does "Collections.singletonMap" really make the code more readable than "singletonMap"?